app/util/path-builder.ts

```diff
@@ -54,20 +54,20 @@ export const pb = {
 
   serialConsole: (params: Instance) => `${pb.instance(params)}/serial-console`,
 
-  diskNew: (params: Project) => `${pb.project(params)}/disks-new`,
+  disksNew: (params: Project) => `${pb.project(params)}/disks-new`,
   disks: (params: Project) => `${pb.project(params)}/disks`,
 
-  snapshotNew: (params: Project) => `${pb.project(params)}/snapshots-new`,
+  snapshotsNew: (params: Project) => `${pb.project(params)}/snapshots-new`,
   snapshots: (params: Project) => `${pb.project(params)}/snapshots`,
-  snapshotImageCreate: (params: Snapshot) =>
-    `${pb.project(params)}/snapshots/${params.snapshot}/image-new`,
+  snapshotImagesNew: (params: Snapshot) =>
+    `${pb.project(params)}/snapshots/${params.snapshot}/images-new`,
```

Comment:
On this one (only this one) I actually changed the route too because it was wrong.